tests/models/unit_cases_electricity/electrolyzer/src/example.py

```diff
@@ -58,6 +58,22 @@ def function(self, optimization_problem, ensemble_member):
         return -sum
 
 
+class MaxH2Goal(Goal):
```

Comment:
Note that this goal is only left there for easy future debugging

Reply:
@jimrojerTNO I would add this in code commenting then. Also state: DO NOT DELETE because .....